### ISSUE TITLE
[question] CLI command to discover the cmake_find_package filename from the Conan package name?

### ISSUE THREAD

**Issue by thejohnfreeman** [conan-io/conan#7525]
The `cmake_find_package` generator writes a Find Module file that defines a target in a namespace. Each of these three names (filename, namespace, target) are [configurable](https://github.com/conan-io/conan/issues/7254) in `cpp_info`. For example,

```python
        self.cpp_info.filename['cmake_find_package'] = 'BoostFile'
        self.cpp_info.names['cmake_find_package'] = 'BoostNamespace'
        self.cpp_info.name = 'BoostTarget'
```

will generate a file named `FindBoostFile.cmake` that is used like this:

```cmake
find_package(BoostFile)
target_link_libraries(${target} BoostNamespace::BoostTarget)
```

Is there a CLI command that can print out these values given a package reference? Something like

```
$ conan inspect -a cpp_info.names.cmake_find_package boost/1.73.0@
```

**Comment by memsharded** [comment]
I think this should be updated in the ``markdown`` generator: https://docs.conan.io/en/latest/reference/generators/markdown.html
(up to my knowledge this ``markdown`` generator has not been updated yet to account for these changes)

Using it would be:

```
$ conan install boost/1.73.0@ -g=markdown
```
And you get the file with the instructions for the different generators.

**Comment by thejohnfreeman** [comment]
That's ok for a human, but I'm looking for something that can be easily consumed by a script.

**Comment by memsharded** [comment]
Ok, lets add that data to the ``json`` generator, something like from:

```json
  "dependencies": [
    {
      "version": "1.2.11",

      "exelinkflags": [],
      "frameworks": [],
      "framework_paths": [],
      "cppflags": [],
      "name": "zlib"
    }
  ],
```
Add:
```json
  "dependencies": [
    {
      "version": "1.2.11",

      "exelinkflags": [],
      "frameworks": [],
      "framework_paths": [],
      "cppflags": [],
      "name": "zlib"
      "names": {"cmake_find_package": "xxxx", "pkg_config": "yyyy"}
      "filenames": {"cmake_find_package": "zzzz"}
    }
  ],
```

**Comment by memsharded** [comment]
This has been added in https://github.com/conan-io/conan/pull/7690 and will be released in Conan 1.30
